src/ngraph/runtime/cpu/pass/cpu_mat_fusion.cpp

```diff
@@ -459,7 +460,7 @@ std::shared_ptr<Node> fuse_batch_mat_mul_transpose(const std::shared_ptr<Node>&
             fuse_input[0], fuse_input[1], transpose[0], transpose[1]);
     }
     return {nullptr};
-}
+}*/
```

Comment:
Why keep a copy of this function here? Also line 476-480.

Reply:
yes, i will remove this from here.